main.scss

```diff
@@ -0,0 +1,14 @@
+@if ($o-normalise-is-silent == false) {
+	@include oNormaliseHTML;
+
+	.o-normalise-clearfix {
+		@include oNormaliseClearfix;
+	}
+
+	.o-normalise-visually-hidden {
+		@include oNormaliseVisuallyHidden;
+	}
+
```

Comment:
Because it's being included in the `oNormaliseHTML` mixin by default. Also this is just for the `non-silent` mode as well